tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -115,6 +115,9 @@ def arithmetic(
 
 
 def test_power(fieldview_backend):
```

Comment:
Should we delete these tests?

Reply:
No, but I can move them to `test_math_builtin_execution.py`. The tests here test the `base ** exp` syntax, while the other tests only test the direct call to `power`.